MockObjects/Objects/MockObjectsFiles/TestMessagesOptions.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "Cryptography.hpp"
+
+namespace TestMessagesOptions
+{
+const std::string testLogin{"memorisecodead"};
```

Comment:
Those are still hardcoded, how does this differ from what we're trying to leave behind?